src/Microsoft.DotNet.ProjectModel/ProjectReader.cs

```diff
@@ -229,6 +229,11 @@ public Project ReadProject(Stream stream, string projectName, string projectPath
 
                 project.RawRuntimeOptions = runtimeOptions.ToString();
             }
+            else if (File.Exists(Path.Combine(projectPath, $"{projectName}.runtimeconfig.json")))
```

Comment:
So, if this file exists, we take the runtime options from it instead of from the project.json? If so, it would be worth adding a test stating that.